2.1/Dockerfile

```diff
@@ -47,6 +54,10 @@ ENV CASSANDRA_VERSION 2.1.19
 
 RUN set -ex; \
 	\
+# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=877677
```

Comment:
This can be just https://bugs.debian.org/877677 (which is nicer to look at IMO)

Reply:
ok